src/overlay/popup.jsx

```diff
@@ -24,6 +24,10 @@ export default class Popup extends Component {
          * 触发弹层显示或隐藏的操作类型，可以是 'click'，'hover'，'focus'，或者它们组成的数组，如 ['hover', 'focus']
          */
         triggerType: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
+        triggerClickKeycode: PropTypes.oneOfType([
```

Comment:
you can add this before line 27:
```js
/**
 * 当 triggerType 为 click 时才生效，可自定义触发弹层显示的键盘码
 */	
```

after adding this, run `npm run api overlay`, it will generate Chinese docs automatically. And then you can add English docs by hand.

https://github.com/alibaba-fusion/next/wiki/PR%E5%89%8D%E8%A6%81%E5%81%9A%E5%88%B0%E7%9A%847%E4%BB%B6%E4%BA%8B-Checklist-Before-PR

Reply:
@youluna added api for chinese and english